src/main/java/com/thebois/persistence/AbstractPersistence.java

```diff
@@ -0,0 +1,33 @@
+package com.thebois.persistence;
+
+import java.io.File;
+
+/**
+ * Basic foundation for persistence, common functionality for all persistence classes.
+ */
+abstract class AbstractPersistence {
+
+    private final String directory =
+        System.getProperty("user.home") + "/Documents/Pawntastic/saves/";
+
+    /**
+     * All subclasses need to use this in the constructor to ensure the directory is there before
+     * any commands/operations can be carried out.
+     */
+    protected void directoryExist() {
```

Comment:
style

If it returned a boolean, this name would fit. Since it actually creates a directory if it does not exist, something along the lines of `ensureSaveDirectoryExists` fits better.